examples/python-operator-dataflow/run.sh

```diff
@@ -4,6 +4,7 @@ python3 -m venv .env
 . $(pwd)/.env/bin/activate
 # Dev dependencies
 pip install maturin
+pip install patchelf
```

Comment:
This addition just fix some linking issue that can happen when people tries the python operator example